Dwolla.Client.Tests/DwollaClientShould.cs

```diff
@@ -166,25 +158,25 @@ private static HttpRequestMessage CreateRequest(HttpMethod method)
             };
 
         private void SetupForGet(HttpRequestMessage req, RestResponse<TestResponse> res) =>
-            _restClient.Setup(x => x.SendAsync<TestResponse>(It.IsAny<HttpRequestMessage>()))
-                .Callback<HttpRequestMessage>(y => GetCallback(req, y)).ReturnsAsync(res);
+            _restClient.Setup(x => x.SendAsync<TestResponse>(It.IsAny<HttpRequestMessage>(), It.IsAny<HttpClient>()))
+                .Callback<HttpRequestMessage, HttpClient>((y, c) => GetCallback(req, y)).ReturnsAsync(res);
 
         private void SetupForPost<T>(HttpRequestMessage req, RestResponse<T> res) =>
-            _restClient.Setup(x => x.SendAsync<T>(It.IsAny<HttpRequestMessage>()))
-                .Callback<HttpRequestMessage>(y => PostCallback(req, y)).ReturnsAsync(res);
+            _restClient.Setup(x => x.SendAsync<T>(It.IsAny<HttpRequestMessage>(), It.IsAny<HttpClient>()))
+                .Callback<HttpRequestMessage, HttpClient>((y, c) => PostCallback(req, y)).ReturnsAsync(res);
 
         private void SetupForUpload(HttpRequestMessage r, RestResponse<EmptyResponse> response) =>
-            _restClient.Setup(x => x.SendAsync<EmptyResponse>(It.IsAny<HttpRequestMessage>()))
-                .Callback<HttpRequestMessage>(y => UploadCallback(r, y)).ReturnsAsync(response);
+            _restClient.Setup(x => x.SendAsync<EmptyResponse>(It.IsAny<HttpRequestMessage>(), It.IsAny<HttpClient>()))
+                .Callback<HttpRequestMessage, HttpClient>((y, c) => UploadCallback(r, y)).ReturnsAsync(response);
 
         private void SetupForDelete(HttpRequestMessage req, RestResponse<EmptyResponse> res) =>
-            _restClient.Setup(x => x.SendAsync<EmptyResponse>(It.IsAny<HttpRequestMessage>()))
-                .Callback<HttpRequestMessage>(y => DeleteCallback(req, y)).ReturnsAsync(res);
+            _restClient.Setup(x => x.SendAsync<EmptyResponse>(It.IsAny<HttpRequestMessage>(), It.IsAny<HttpClient>()))
+                .Callback<HttpRequestMessage, HttpClient>((y, c) => DeleteCallback(req, y)).ReturnsAsync(res);
 
         private static async void PostCallback(HttpRequestMessage expected, HttpRequestMessage actual)
         {
             GetCallback(expected, actual);
-            Assert.Equal("{\"message\":\"requestTest\"}", await actual.Content.ReadAsStringAsync());
+            Assert.Equal("{\"Message\":\"requestTest\"}", await actual.Content.ReadAsStringAsync());
```

Comment:
Can we replace `Message` with `message`? It seems to fail the test, `Dwolla.Client.Tests.DwollaClientShould.CreatePostRequestAndPassToClient`, where it expects `Message` but gets a lowercase `message`.
```suggestion
            Assert.Equal("{\"message\":\"requestTest\"}", await actual.Content.ReadAsStringAsync());
```

Reply:
Yup! Not sure how I missed that